content/_guides/managing-user-roles-in-a-tenant.md

```diff
@@ -1,26 +1,19 @@
 ---
 title: Managing user roles in a contract
-description: This document explains what roles and permissions are, how roles are set in the UI and how custom roles are configured.
+description: This document explains the default roles and permissions of the platform and how to set them in the UI.
 layout: article
 section: Contract Management
 order: 1
 since: 20190321
 category: integrator-management
-redirect_from:
-  - /guides/managing-user-roles-in-a-tenant#setting-user-roles.html
-  - /guides/managing-user-roles-in-a-tenant#configuring-custom-user-roles.html
-  - /guides/managing-user-roles-in-a-tenant#essential-roles.html
-  - /guides/managing-user-roles-in-a-tenant#permissions-reference-table.html
 ---
 
-This document explains [what roles and permissions are](#roles-and-permissions)
-and how [to set roles in the UI](#setting-user-roles).
+{{page.description}}
 
 ## Roles and permissions
 
-A tenant admin can
-control user rights in his tenant by configuring user roles. A role is a user
-attribute that defines a set of permissions for the user. Contract roles define
+A tenant admin can control user rights in his tenant by configuring user roles.
```

Comment:
his/her?

Reply:
I think I would prefer to write not his or her but "tenant admin can control user rights in a tenant"